CHANGELOG.md

```diff
@@ -53,6 +53,22 @@ v2.13.alpha (???) - (??/??/????)
 		- SET_GLOBAL_SHIFT {x} {y} {z} -KEEP_ORIG_FIXED
 			- set global shift on all entities
 			- sub-option -KEEP_ORIG_FIXED: if set, global origin will be preserved (a warning might be issued if the resulting coordinate transformation is too big)
+		- SELECT_ENTITIES -{FIRST/LAST} {nr} {-ALL} {-NOT} {-REGEX} {regex_pattern}
+			-sub-option -FIRST {nr}: if set, first nr of entities will be selected (optional). example: xx.....
+			-sub-option -LAST {nr}: if set, last nr of entities will be selected (optional). example: .....xx
+			-sub-option -ALL: if set, all entities will be selected. It has higher priority than FIRST/LAST/REGEX. (optional). example: xxxxxxx
+			-sub-option -REGEX {regex_pattern}: if set, all entities with matching names will be selected. It has higher priority than FIRST/LAST. (optional). example: x(merged)..x(merged)...
+			-sub-option -NOT: if set, all condition will be reversed (optional)
+				- first {nr} -> except first {nr} (optional). example: ..xxxxx
+				- last {nr} -> except last {nr} (optional). example: xxxxx..
+				- all -> none (optional). example: .......
+				- regex matched -> regex not matched (optional). example: .(merged)xx.(merged)xxx
+				- special case: first and last set -> all except first {nr} AND last {nr}. example: ..xxx..
+			-sub-option -CLOUD: if set, clouds will be affected (optional)
```

Comment:
Well if you pass.
-CLOUD only cloud vector will be affected leave m_meshes as is
-MESH only mesh vector will be affected leave m_clouds as is

But the user can pass both of them
-CLOUD -MESH in that case both will run similar when both of them omitted

Reply:
Yes, it's just the wording that was strange ('affected' has a negative/psychological connotation). And the command is 'SELECT' anyway 😉

Reply:
maybe i misunderstood should i make the command line option to -SELECT instead of -SELECT_ENTITIES, or just replace the affected wording in the changloeg with selected.

It is shorter, and the additional options defines the entity type. I'll need to check if there is any command collision

Reply:
No, SELECT_ENTITIES is good (I was just lazy). You should mainly change 'affected' by 'selected' indeed.